pkg/agent/types/multicast.go

```diff
@@ -35,9 +35,8 @@ var (
 	_, McastCIDR, _ = net.ParseCIDR("224.0.0.0/4")
 )
 
-type MulticastValidator interface {
-	// Validate checks whether IGMP report from Pod(podNamespace/podName) to groupAddress should be dropped,
+type McastNetworkPolicyController interface {
+	// GetIGMPNPRuleInfo looks up if there is any rule matches to a Pod(podNamespace/podName) to groupAddress,
```

Comment:
looks up the IGMP NetworkPolicy rule that matches the given Pod and groupAddress, and returns the rule information if found.

Reply:
done